test/web_console/railtie_test.rb

```diff
@@ -45,7 +46,16 @@ class RailtieTest < ActiveSupport::TestCase
         app.config.web_console.mount_point = '/customized/path'
         app.initialize!
 
-        assert_equal Middleware.mount_point, '/customized/path'
+        assert_equal '/customized/path', Middleware.mount_point
+      end
+    end
+
+    test 'config.mount_point supports relative_url_root' do
+      new_uninitialized_app do |app|
+        app.config.relative_url_root = '/relative/path'
```

Comment:
Can you also add a test where both `app.config.relative_url_root` and `app.web_console.mount_point` are set?